backend/src/db/migrations/20240205200732_invite-only.ts

```diff
@@ -0,0 +1,27 @@
+import { Knex } from "knex";
+
+import { TableName } from "../schemas";
+
+export async function up(knex: Knex): Promise<void> {
+  const isTablePresent = await knex.schema.hasTable(TableName.SuperAdmin);
+  if (isTablePresent) {
+    await knex.schema.alterTable(TableName.SuperAdmin, (t) => {
+      t.boolean("inviteOnlySignUp").defaultTo(false);
+      t.string("allowSpecificDomainSignUp");
+    });
+  }
+}
+
+export async function down(knex: Knex): Promise<void> {
+  if (await knex.schema.hasColumn(TableName.SuperAdmin, "inviteOnlySignUp")) {
+    await knex.schema.alterTable(TableName.SuperAdmin, (t) => {
+      t.dropColumn("inviteOnlySignUp");
+    });
+  }
+
+  if (await knex.schema.hasColumn(TableName.SuperAdmin, "allowSpecificDomainSignUp")) {
+    await knex.schema.alterTable(TableName.SuperAdmin, (t) => {
+      t.dropColumn("allowSpecificDomainSignUp");
```

Comment:
`allowSpecificDomainSignUp` this feels as if this is boolean. I feel its bettter to name it as `allowedSignupDomain`.

specific also doesn't make sense.

Reply:
CC: @maidul98

Reply:
Agree, `allowedSignupDomain` makes more sense

Reply:
Done